sql/sql_view.cc

```diff
@@ -455,6 +453,8 @@ bool mysql_create_view(THD *thd, TABLE_LIST *views,
     goto err_no_relink;
   }
 
+  WSREP_TO_ISOLATION_BEGIN(WSREP_MYSQL_DB, NULL, NULL);
+
```

Comment:
TOI replication should happen before open_and_lock_tables(), so better move this back to original position

Reply:
To handle timeouts during open_and_lock_tables(), we should disable timeouts for TOI replicated sessions